### PR TITLE
Add more entries to subjectAltName on pulp-certs

### PR DESCRIPTION
**Pull request by elyezer**
This should fix the failures related to Docker cli on Pulp Automation Jobs.

### CODE REVIEW

File: ci/ansible/roles/pulp-certs/tasks/main.yml [hunk 1]
```diff
@@ -20,13 +20,13 @@
 - name: Configure openssl subjectAltName
   lineinfile: dest=/etc/pki/tls/openssl.cnf
               insertafter="^\[ usr_cert \]"
-              line="subjectAltName=DNS:{{ansible_fqdn}}"
+              line="subjectAltName=DNS:{{ ansible_fqdn }},DNS:{{ ansible_nodename }},DNS:{{ ansible_hostname }}"
```
Comment (jeremycline) [inline ci/ansible/roles/pulp-certs/tasks/main.yml]: This looks reasonable to me.

Out of curiosity, do you know what the difference might be between the nodename and the fqdn? In my brief search of the Ansible documentation I wasn't able to find out the exact difference.

Comment (elyezer) [inline ci/ansible/roles/pulp-certs/tasks/main.yml]: I am not sure the real difference and the source of that information but on a system I was trying I got:

```
        "ansible_nodename": "rhel7-vanilla-np-qeos-88822.slave.openstack.org.novalocal",
        "ansible_fqdn": "rhel7-vanilla-np-qeos-88822.novalocal",
```

Seems that `ansible_fqdn` is `ansible_hostname.ansible_domain` and `ansible_nodename` is what `hostname` command reports.

Comment (jeremycline) [inline ci/ansible/roles/pulp-certs/tasks/main.yml]: Oh, weird

**Comment by elyezer** [comment]
Let's merge and see what happens.

**Pull request merged**